tests/test_api_v1.py

```
@@ -279,9 +280,9 @@ def test_submit_handler(ctx):
     assert expected == resp.data
     assert "200 OK" == resp.status
     query = tables.submission.select().order_by(tables.submission.c.id.desc()).limit(1)
-    submission = ctx.db.get_ingest_db().execute(query).fetchone()
-    assert "b9c05616-1874-4d5d-b30e-6b959c922d28" == submission["mbid"]
-    assert "FLAC" == submission["format"]
-    assert 192 == submission["bitrate"]
-    assert TEST_1_FP_RAW == submission["fingerprint"]
-    assert TEST_1_LENGTH == submission["length"]
+    submission = ctx.db.get_ingest_db().execute(query).one()
+    assert UUID("b9c05616-1874-4d5d-b30e-6b959c922d28") == submission.mbid
+    assert "FLAC" == submission.format
```

Ruff (0.8.2)

Yoda condition detected

Rewrite as `submission.format == "FLAC"`

(SIM300)